esx_service/vmdk_ops.py

```diff
@@ -1469,6 +1473,15 @@ def execRequestThread(client_socket, cartel, request):
             reply_string = {u'Error': "Failed to parse json '%s'." % request}
             send_vmci_reply(client_socket, reply_string)
         else:
+            logging.debug("execRequestThread: req=%s", req)
+            version = req["version"]
+            if version != VMCI_VERSION:
```

Comment:
This is too strict expecting user to have exact same version of plugin and esx service. 

I think we need "MIN_SUPPORTED_VERSION" and "MAX_SUPPORTED_VERSION" and need to maintain forward/backward compatibility for at lease 1 release. For e.g. for upcoming release, 0.13 Plugin should be compatible with 0.12 Server or vice-versa.